genie-web/src/main/java/com/netflix/genie/web/rpc/grpc/services/impl/v4/GRpcJobServiceImpl.java

```diff
@@ -34,29 +34,29 @@
 import java.util.UUID;
 
 /**
- * Extension of {@link com.netflix.genie.proto.JobSpecificationServiceGrpc.JobSpecificationServiceImplBase} to provide
+ * Extension of {@link com.netflix.genie.proto.JobServiceGrpc.JobServiceImplBase} to provide
  * functionality for resolving and fetching specifications for jobs to be run by the Genie Agent.
  *
  * @author tgianos
  * @since 4.0.0
  */
 @GrpcService(
-    value = JobSpecificationServiceGrpc.class,
+    value = JobServiceGrpc.class,
     interceptors = {
         SimpleLoggingInterceptor.class,
     }
 )
 @Slf4j
-public class GRpcJobSpecificationServiceImpl extends JobSpecificationServiceGrpc.JobSpecificationServiceImplBase {
+public class GRpcJobServiceImpl extends JobServiceGrpc.JobServiceImplBase {
 
     private final JobSpecificationService jobSpecificationService;
 
     /**
      * Constructor.
-     *
+     *GRpcJobServiceImplSpec
```

Comment:
Stray line?

Reply:
yeah probably something while i was debugging and got in here due to a pasting in the wrong spot. I'll remove it